[request.py]
  [test_continue_auth_required[fetch\]]
    expected: FAIL

  [test_continue_auth_required[navigate\]]
    expected: FAIL

  [test_continue_response_started[fetch\]]
    expected: FAIL

  [test_continue_response_started[navigate\]]
    expected: FAIL
